### PR TITLE
feat: for crawler, export node addresses (w/o port) instead of hash ids

### PR DESCRIPTION
**Pull request by kylegranger**

### CODE REVIEW

File: src/tools/crawler/metrics.rs [hunk 1]
```diff
@@ -46,7 +46,7 @@ pub struct NetworkSummary {
     protocol_versions: HashMap<u32, usize>,
     user_agents: HashMap<String, usize>,
     crawler_runtime: Duration,
-    node_ids: Vec<String>,
+    node_addrs: Vec<String>,
```
Comment (ljedrz) [inline src/tools/crawler/metrics.rs:49]: if these are just IPs, I'd name this field `node_ips` so as not to mix them up with `SocketAddr` which contains the port too

Comment (kylegranger) [reply src/tools/crawler/metrics.rs]: done

File: src/tools/crawler/metrics.rs [hunk 2]
```diff
@@ -67,14 +67,18 @@ impl NetworkSummary {
 
         let num_good_nodes = good_nodes.len();
         let good_addresses: Vec<SocketAddr> = good_nodes.keys().cloned().collect();
-        let mut node_ids: Vec<String> = Vec::new();
+        let mut node_addrs: Vec<String> = Vec::new();
+
+        // remove colon and port number
+        let re = Regex::new(r"(.*):").unwrap();
```
[the remark this answers is not in the document]
Comment (kylegranger) [reply src/tools/crawler/metrics.rs]: FTW  🚀  (done)

**Review by ljedrz — APPROVED** [review]
LGTM :ok_hand:

**Pull request merged**